### PR TITLE
Update prow config for serving-operator to use knative.dev

### PR DESCRIPTION
**Pull request by trshafer**
**What this PR does, why we need it**:

**Which issue(s) this PR fixes**:
Addresses Prow issues in https://github.com/knative/serving-operator/pull/68

**Special notes to reviewers**:

**User-visible changes in this PR**:

### CODE REVIEW

**Comment by chaodaiG** [comment]
you need to change this file instead: https://github.com/knative/test-infra/pull/1048/files#diff-6b96213e43367b89ca71567c589995ed

The `config.yaml` is generated by `make config` under `ci/prow`

**Comment by chaodaiG** [comment]
/hold
will need to coordinate with this PR: https://github.com/knative/serving-operator/pull/68
@trshafer  please dm me when you're ready to merge both PRs

**Comment by trshafer** [comment]
@chaodaiG updated and ran `make config` within `ci/prow`.

File: ci/prow/config_knative.yaml [hunk 1]
```diff
@@ -269,15 +273,19 @@ periodics:
   knative/sample-controller:
     - continuous: true
       dot-dev: true
-  
+
```
Comment (chaodaiG) [inline ci/prow/config_knative.yaml]: thanks for fixing this

**Comment by chaodaiG** [comment]
/lgtm
/approve

Let's remove the `/hold` when we are ready to merge https://github.com/knative/serving-operator/pull/68

**Pull request merged**